### PR TITLE
Add a section on visual diff testing to DEVELOPING.md

### LINKED ISSUE #10540: Add info about percy in contributors guide
We should 

- explain how people can get access to percy (for now: google form to submit email?)
- what to do if there is a failure.

### PR DESCRIPTION
**Pull request by rsimha-amp**
Fixes #10540

### CODE REVIEW

**Comment by rsimha-amp** [comment]
@cramforce, this is ready to merge. I'll hold off for some time in case you have any comments.

File: contributing/DEVELOPING.md [hunk 1]
```diff
@@ -182,6 +182,43 @@ To run the tests on Sauce Labs:
...
+
+The technology stack used is:
+
+- [Percy](https://percy.io/), a visual regression testing service for webpages
+- [Capybara](https://percy.io/docs/clients/ruby/capybara-rails), a framework that integrates tests with Percy
+- [Poltergeist](https://github.com/teampoltergeist/poltergeist), a driver capable of loading webpages for diffing
+- [PhantomJS](http://phantomjs.org/), a headless webkit based browser
+
+The [`ampproject/amphtml`](https://github.com/ampproject/amphtml) repository on GitHub is linked to the [Percy project](https://percy.io/ampproject/amphtml) of the same name. All PRs will show a check called `percy/amphtml` in addition to the `continuous-integration/travis-ci/pr` check. If your PR results in visual diff(s), clicking on the `details` link will show you the snapshots with the diffs highlighted. (If you don't have permissions to view the diffs, you can make a request [here](https://docs.google.com/forms/d/e/1FAIpQLScZma6qVJtYUTqSm4KtiF3Zc-n5ukNe2GXNFqnaHxospsz0sQ/viewform).)
```
Comment (cramforce) [inline contributing/DEVELOPING.md]: Please hoist the last `(…)` to the beginning and explain/apologize why the process is needed and that we will grant access.

Comment (rsimha-amp) [inline contributing/DEVELOPING.md]: Done.

**Comment by rsimha-amp** [comment]
All comments addressed. Merging this now.

**Pull request merged**